sript.js

```diff
@@ -196,4 +193,47 @@ form.addEventListener('submit', (event) => {
     email.classList.add('error-shadow');
     error.style.display = 'block';
   }
-});
\ No newline at end of file
+});
+
+// Keep Data
+
+const fullName = document.getElementById('name');
+const comment = document.getElementById('comment');
+
+function keep() {
+  const input = {
+    inputName: fullName.value,
+    inputEmail: email.value,
+    inputComment: comment.value,
+  };
+  localStorage.setItem('input', JSON.stringify(input));
+}
+
+let formObject = JSON.parse(localStorage.getItem('input'));
+
+if (formObject === undefined) {
+  formObject = {
+    fullName: '',
+    email: '',
+    comment: '',
+  };
+  keep();
+}
+
+fullName.value = formObject.fullName;
+fullName.addEventListener('change', (event) => {
+  formObject.fullName = event.target.value;
+  localStorage.setItem('input', JSON.stringify(formObject));
+});
+
+email.value = formObject.email;
+email.addEventListener('change', (event) => {
+  formObject.email = event.target.value;
+  localStorage.setItem('input', JSON.stringify(formObject));
+});
+
+comment.value = formObject.comment;
+comment.addEventListener('change', (event) => {
+  formObject.comment = event.target.value;
+  localStorage.setItem('input', JSON.stringify(formObject));
+});
```

Comment:
Kindly implement the changes suggested by the previous reviewer, as your data is still not saved in the local storage. please try to fix the problem before submitting for another review 😄 

Current behaviour :

https://github.com/Arch-Noize/portfolio-microverse/assets/20537712/f28e138a-797e-42ba-bda3-851d8e1ed0d5

Reply:
Hi, I... did implement those changes however. The code is working just fine on my end. Previously in another project I had a similar weird issue where my reviewers couldn't see my changes be displayed in their end, but could be actually seen locally, for whatever reason

Reply:
Sorry to be of bother, I am not entirely sure what's happening here

Reply:
I'm adding a video of my own showing it is working on my end. I don't know what's going on

https://github.com/Arch-Noize/portfolio-microverse/assets/91759364/5ad01d91-69d2-41b1-a1e1-b706d450ccc7